ui/src/message_popup/message_popup_page.jsx

```diff
@@ -147,8 +151,8 @@ export default React.createClass({
             <Toggle
               label="With hints available"
               labelPosition="right"
-              toggled={false}
-              disabled={true} />
+              toggled={this.state.allowedToToggleHint}
+              onToggle={this.onHintsOptionToggled} />
```

Comment:
We'll have to pass `this.state.allowedToToggleHint` in to the `PopupQuestion` component too, won't we?  That way if we can use it [here](https://github.com/mit-teaching-systems-lab/threeflows/pull/20/files#diff-49aaf0f1ebf22819c85f8d7790ddbb18R85) and we can hide the "Show hint" button if it's false.